packages/server/lib/hooks/hooks.ts

```diff
@@ -187,17 +190,21 @@ export const connectionRefreshSuccess = async ({
     connection: Pick<DBConnectionDecrypted, 'id' | 'connection_id' | 'provider_config_key' | 'environment_id'>;
     config: IntegrationConfig;
 }): Promise<void> => {
-    await errorNotificationService.auth.clear({
-        connection_id: connection.id
-    });
+    try {
+        await errorNotificationService.auth.clear({
+            connection_id: connection.id
+        });
 
-    await slackService.removeFailingConnection({
-        connection,
-        name: connection.connection_id,
-        type: 'auth',
-        originalActivityLogId: null,
-        provider: config.provider
-    });
+        await slackService.removeFailingConnection({
+            connection,
+            name: connection.connection_id,
+            type: 'auth',
+            originalActivityLogId: null,
+            provider: config.provider
+        });
```

Comment:
I agree that `auth.clear` and `slack.removeFailingConnection` are secondary. Should they have their own try catch though because here if auth.clear fails the slack notif will not be attempted

Reply:
true

Reply:
changed